src/sentry/static/sentry/app/views/settings/components/settingsPluginsNavigation.jsx

```diff
@@ -22,8 +22,9 @@ class SettingsPluginsNavigation extends React.Component {
     ...SentryTypes.NavigationGroup,
     organization: SentryTypes.Organization,
     project: SentryTypes.Project,
-    access: PropTypes.object,
-    features: PropTypes.object,
+
+    access: PropTypes.object, // eslint-disable-line react/no-unused-prop-types
+    features: PropTypes.object, // eslint-disable-line react/no-unused-prop-types
```

Comment:
Both `access` and `features` don't appear to be used? Why do we need them?

Reply:
Ah you're right, I thought this was the same thing as the other one. Yeah I'll remove.

Reply:
Fixed.